Tests/Core/Networking/RequestExecutorTests.swift

```diff
@@ -0,0 +1,422 @@
+import XCTest
+
+@testable import Frames
+
+final class RequestExecutorTests: XCTestCase {
```

Comment:
lots of the tests in this class look similar, would it make sense to build some parameterised tests?

Reply:
I suspect that parameterising these tests could make the structure quite complex, but I'd be happy to consider any suggestions 😄

Reply:
Some of them obviously fall outside of this but generally it looks like you do a few things:

1. Setup `stubRequestParameterProvider`
1. Setup `responseType` (always constant)
1. Call execute with a closure
    - This completion handler could check the args or calls to stubs

so you could make a dictionary of these arguments and make a test that goes through the dictionary